### DISCUSSION TITLE
Ellipsis for overflow text in dropdown boxes

### DISCUSSION THREAD

**Discussion by liemnd161** [sagalbot/vue-select#1376] [Q&A]
Hi,
As i checked with the demo the vue-select will show full selected option.
![image](https://user-images.githubusercontent.com/30147027/108798406-b7a6c100-75c8-11eb-91e7-90e309421a57.png)

Does it have any option that can show selected option with ellipsis in case the text is too long? Or i have to custom css?

**Comment by exartt** [comment]
I'm using like this: 
```
<template #option="{ label }">
    <div style="display: block; width: 100%; white-space: nowrap; overflow: hidden; text-overflow: ellipsis;" >
      {{ label }}
    </div>
</template>
```
